docs/admin-manual/ap-policy-configuration.rst

```diff
@@ -571,3 +571,85 @@ See the :ref:`admin-manual/configuration-macros:condor_master configuration
 file macros` section for details relating to the configuration variables used
 to set timing and polling intervals.
 
+Performance Tuning of the AP
+----------------------------
+
+Of the three roles (AP, CM, EP) in a HTCondor system, the AP is the most common
+place performance tuning is done.  The CM is mostly stateless, and can
+typically scale out to very large pools without much additional work.  The EP
+daemons aren't resource intensive.  However, as the EP stores the state of all
+the jobs under its control, and persistently stores frequent updates to those
+jobs, it is not uncommon for the AP to exhaust system resources, like cpu, or
+disk and network bandwidth.
```

Comment:
Starts talking about EP and ends with AP... is this supposed to say `However, as the AP...`

Reply:
Oops. typo.  Fixed.